package.json

```diff
@@ -2,9 +2,10 @@
   "name": "suomifi-ui-components",
   "version": "6.1.0-beta.0",
   "description": "Suomi.fi UI component library",
-  "main": "dist/index.js",
-  "module": "dist/suomifi-ui-components.esm.js",
-  "typings": "dist/suomifi-ui-components.esm.d.ts",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "typings": "dist/types/index.d.ts",
```

Comment:
Tested to build locally, and it seems that .d.ts file/files does no longer exist in the builds at the moment.

Reply:
This should be now fixed with the f602fd566095a6babc499d6aaba0bd8d83f21073 commit.
Was missing a script to copy the types to the **dist** directory